src/main/java/com/ea/gatling/GatlingAwsMojo.java

```diff
@@ -54,6 +54,9 @@
     @Parameter(property = "ec2.force.termination", defaultValue = "false")
     private boolean ec2ForceTermination = false;
 
+    @Parameter(property = "ec2.keep.alive", defaultValue="false")
+    private boolean ec2KeepAlive = false;
```

Comment:
Do you mind adding a comment to the new parameter? Please also document it in the README.